tex/measure/swapsum.tex

```diff
@@ -43,20 +43,65 @@ \section{Motivating limit interchange}
 	We can define $\liminf_{n \to \infty} f_n$
 	and $\limsup_{n \to \infty} f_n$ similarly.
 \end{definition}
-This is actually a fairly weak notion of convergence, for example:
-\begin{exercise}
-	[Witch's hat]
-	Find a sequence of continuous function on $[-1,1] \to \RR$
-	which converges pointwise to the function $f$ given by
-	\[ f(x) = \begin{cases}
-			1 & x = 0 \\
-			0 & \text{otherwise}.
-		\end{cases} \]
-\end{exercise}
-This is why when thinking about the Riemann integral
-it is commonplace to work with stronger conditions like
-``uniformly convergent'' and the like.
-However, with the Lebesgue integral, we can mostly not think about these!
+
+By ``the Lebesgue integral has better behavior'', we means the following:
```

Comment:
I'm... pretty sure before the change, the book misses the *point* of the convergence theorems.

The "better behavior" described in this section is actually a feature of *measurable* functions (i.e. lim sup and lim inf of a sequence of measurable functions is still measurable, this property is not hold for continuous functions or Riemann integrable functions)

Reply:
sounds correct.